test/commands/files.test.js

```diff
@@ -1785,10 +1791,12 @@ describe('Files', () => {
 				let downloadedFilePath = path.join(fileDownloadPath, fileName);
 				let downloadContent = fs.readFileSync(downloadedFilePath);
 				let expectedContent = fs.readFileSync(testFilePath);
+				let downloadedFileStat = fs.statSync(downloadedFilePath);
+				assert.equal(testFileStat.size, downloadedFileStat.size);
 				fs.unlinkSync(downloadedFilePath);
 				/* eslint-enable no-sync */
 				assert.ok(downloadContent.equals(expectedContent));
-				let expectedMessage = getDownloadProgressBar(295191);
+				let expectedMessage = getDownloadProgressBar(testFileStat.size);
```

Comment:
Changing line endings from `CLRF` to `LF` for the `epic-poem.txt` fixes this test for me locally without your changes. Could that be the issue? Maybe the repo was cloned with windows line endings?

Anyway, I think your approach is better as it should work regardless of the line endings, but we should somehow explain why it's done this way as it's not obvious from the code (either by extracting some parts to a named function or leaving a comment at least)